dso-l2/src/main/java/com/tc/l2/state/ConsistencyMBeanImpl.java

```diff
@@ -49,6 +51,7 @@ public boolean isStuck() {
 
   @Override
   public void allowRequestedTransition() {
+    GuardianContext.validate(Guardian.Op.STATE_CHANGE, "Promote invoked");
     consistencyManager.allowLastTransition();
```

Comment:
I understand - I was trying to have IDs that contain exactly what we want to audit, so that we don't have to speical-case a bunch of things. This isn't right, I know. Could we add a new `message` parameter in the `Guardian::validate` method so that I could achieve this?

In addition to adding a new parameter to `Guardian::validate`, I could change "Promote invoked" to something like "Request for server state transition to X state received". How does that sound? Also, what's the easiest way to get the new state X?